app/controllers/webhooks/base_controller.rb

```diff
@@ -0,0 +1,49 @@
+# This is the base controller of Exercism's webhooks.
+#
+# It is intended to only be called from GitHub webhooks
+# and should not be called by any other source
+
+module Webhooks
+  class BaseController < ApplicationController
+    skip_before_action :verify_authenticity_token
+    skip_before_action :authenticate_user!
+
+    before_action :verify_github_webhook
```

Comment:
Throw a `!` on the end of this as it's an explosive action.

Reply:
Bang!